Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM node:18.18-alpine3.18
+# Keep this in sync with the Node version specified for the Conda dev environment (in environment_dev.yml)
+FROM node:22.8.0
```

Comment:
Not sure if removing `alpine` was the right move, but I can confirm that it worked for me when I ran `docker compose --profile local-dev up --build`

Reply:
This changes the base to Debian (see [22.8 on DockerHub](https://hub.docker.com/layers/library/node/22.8.0/images/sha256-eb5bb667442cadcd1bb8e6b3d44b2d11bbc5beb280db7f022872d33177b61ca1?context=explore), which might mean our Docker image is a bit bigger than it used to be but not a major issue tbh)